src/test/static/editor-service/references.html

```diff
@@ -1,16 +1,16 @@
-<html>
-<head>
-<link rel="import" href="../analysis/simple/simple-element.html">
-<script src="../vanilla-elements.js"></script>
-</head>
-<body>
-<div>
-  <anonymous-class></anonymous-class>
-  <simple-element></simple-element>
-  <div>
-    <simple-element></simple-element>
-  </div>
-  <anonymous-class></anonymous-class>
-</div>
-</body>
-</html>
+<html>
+<head>
+<link rel="import" href="../analysis/simple/simple-element.html">
+<script src="../vanilla-elements.js"></script>
+</head>
+<body>
+<div>
+  <anonymous-class></anonymous-class>
+  <simple-element></simple-element>
+  <div>
+    <simple-element></simple-element>
+  </div>
+  <anonymous-class></anonymous-class>
```

Comment:
Minor: recommend adding some attributes to each of these just to make them easy to distinguish in the tests. e.g. (`<analymous-class one>` `<simple-element two>` etc)